### PR TITLE
Move the entry point of the API to the window.navigator interface.

### LINKED ISSUE #67: Move the posture object out of the Screen interface.
Now that the specification only covers the posture it seems weird to have the main entry point of the API to be in the `Screen` interface. 

I strongly think we should move it to another place like `window` or `window.navigator`. The thinking is that there is only a single posture at a given time and it's not dependent on the screen.

What does sound best to you folks?

- `window.navigator.posture`

- `window.posture` which is basically `posture`

I strongly think that it we land into a generic object like `navigator` or `window` we should rename the entry point from `posture` to `device_posture`.

### PR DESCRIPTION
**Pull request by darktears**
Closes #67

***
<a href="https://pr-preview.s3.amazonaws.com/darktears/device-posture/pull/71.html" title="Last updated on Apr 30, 2021, 4:40 PM UTC (a3272e6)">Preview</a> | <a href="https://pr-preview.s3.amazonaws.com/w3c/device-posture/71/fb2d39d...darktears:a3272e6.html" title="Last updated on Apr 30, 2021, 4:40 PM UTC (a3272e6)">Diff</a>

### CODE REVIEW

**Comment by darktears** [comment]
@anssiko @kenchris @beaufortfrancois @lauramorinigo

**Review by kenchris — APPROVED** [review]

File: index.html [hunk 1]
```diff
@@ -640,8 +640,8 @@ <h2>
       <h3>Example 1: Posture data</h3>
       <p>This is a simple use case of the <a>posture</a> being printed on the console. </p>
       <pre class="example js" title="React to poster change">
-          screen.posture.addEventListener("change", () => {
-            console.log(`The current posture is: ${screen.posture.type}!`);
+          window.navigator.devicePosture.addEventListener("change", () => {
```
Comment (beaufortfrancois) [inline index.html]:
```suggestion
          navigator.devicePosture.addEventListener("change", () => {
```

File: index.html [hunk 2]
```diff
@@ -640,8 +640,8 @@ <h2>
...
+            console.log(`The current posture is: ${window.navigator.devicePosture.type}!`);
```
Comment (beaufortfrancois) [inline index.html]:
```suggestion
            console.log(`The current posture is "${navigator.devicePosture.type}".`);

```

**Review by beaufortfrancois — APPROVED** [review]
LGTM with nits

**Pull request merged**